src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -8186,6 +8186,8 @@ public static String strip(String str, final String stripChars) {
         return stripEnd(str, stripChars);
     }
 
+    private static final Pattern STRIP_ACCENTS_PATTERN = Pattern.compile("\\p{InCombiningDiacriticalMarks}+"); //$NON-NLS-1$
```

Comment:
All constants live at the top of the file.

Reply:
@garydgregory 
done.